pyrefinebio/computational_result.py

```diff
@@ -0,0 +1,44 @@
+import pyrefinebio.common.annotation as prb_annotation
+import pyrefinebio.common.organism_index as prb_organism_index
+import pyrefinebio.computed_file as prb_computed_file
+import pyrefinebio.processor as prb_processor
+from pyrefinebio.http import get_by_endpoint
+from pyrefinebio.util import generator_from_pagination
+
+
+class ComputationalResult:
+    def __init__(
+        self,
+        id=None,
+        commands=None,
+        processor=None,
+        is_ccdl=None,
+        annotations=[],
```

Comment:
Careful with this. It looks like you overwrite it anyway, but this can go badly if this parameter ever actually gets used.

See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments for an explanation why.

I think the better thing to do is have lists default to None as well.